django/db/backends/base/schema.py

```diff
@@ -1008,7 +1013,14 @@ def create_fk_name(*args, **kwargs):
     def _delete_fk_sql(self, model, name):
         return self._delete_constraint_sql(self.sql_delete_fk, model, name)
 
-    def _unique_sql(self, fields, name):
+    def _unique_sql(self, model, fields, name, condition=''):
```

Comment:
Should it be `condition=None` or `condition=''` in all signatures (currently some inconsistency)? I'd lean toward the former.

Reply:
Same `condition=''` for indexes: https://github.com/django/django/blob/master/django/db/backends/base/schema.py#L897

Reply:
I like `condition=None`, but I like more consistency with index signature, should I fix both unique_constraint and index signatures?

Reply:
Okay, as before, the other cleanups can be in a separate commit.

Reply:
added as separate commit both for index and unique constraint